NobankCore/noBank/src/main/java/com/hunk/nobank/NoBankApplication.java

```diff
@@ -10,6 +11,9 @@
 	@Override
 	public void onCreate() {			
 		super.onCreate();
+        // Set Logging TAG
+        Logging.TAG = getPackageName();
```

Comment:
Set the TAG when creating Application, in this way we can get the current package name and print it as a TAG.